### PR TITLE
Add service account support

### PR DESCRIPTION
**Pull request by anayden**

### CODE REVIEW

File: .neuro/live.yml [hunk 1]
```diff
@@ -31,17 +31,17 @@ volumes:
```
Comment (YevheniiSemendiak) [inline .neuro/live.yml:31]: what about volumes?

File: README.md [hunk 2]
```diff
@@ -28,10 +30,35 @@ Create a secret with a private SSH key for accessing the GitHub repository (pull
...
+export PROJECT=mlops-demo-oss-dogs
+export PREFIX="/${USER}/${PROJECT}/"
+export ACCOUNT=${USER}/service-accounts/${PROJECT}
+export ROLE=${USER}/projects/${PROJECT//-/_}
+
+neuro acl grant storage:${PREFIX} ${ROLE} write
+neuro acl grant job:/${ACCOUNT} ${ACCOUNT} manage
+neuro acl grant image:${PREFIX} ${ROLE} read 
+neuro acl grant secret:gh-rsa ${ROLE} read
+neuro acl grant role://${ROLE} ${ACCOUNT} read
```
Comment (YevheniiSemendiak) [inline README.md:58]: I think we should create a feature request in NF for doing this thing via a dedicated command.
Something like `neuro-flow rbac enable`, or something else. WDYT?

File: README.md [hunk 3]
```diff
@@ -28,10 +30,35 @@ Create a secret with a private SSH key for accessing the GitHub repository (pull
...
+
+# temp workaround for neuro-flow bug
```
Comment (YevheniiSemendiak) [inline README.md:60]: pls, link the bug here

File: config/pipeline.json [hunk 4]
```diff
@@ -4,16 +4,16 @@
   },
   "description": "Retrain the model after labeling data",
   "transform": {
-    "image": "neuromation/neuro-flow:21.4.5",
+    "image": "ghcr.io/neuro-inc/neuro-flow:21.11.2",
     "cmd": [
       "bash",
       "-c",
-      "neuro config login-with-token ${NEURO_TOKEN}; neuro config switch-cluster ${NEURO_CLUSTER}; cd /tmp; mkdir -p .neuro; echo $BATCH_B64 | base64 -d > .neuro/training.yaml; echo $PARAMS_B64 | base64 -d > .neuro/params.yaml; neuro-flow bake training --meta-from-file .neuro/params.yaml --local-executor"
+      "cd /tmp; mkdir -p .neuro; echo $PROJECT_B64 | base64 -d > .neuro/project.yml; echo $BATCH_B64 | base64 -d > .neuro/training.yaml; echo $PARAMS_B64 | base64 -d > .neuro/params.yaml; neuro-flow --show-traceback bake training --meta-from-file .neuro/params.yaml --local-executor"
```
Comment (YevheniiSemendiak) [inline config/pipeline.json:11]: what if we put it into the `Makefile` ?

**Review by YevheniiSemendiak — COMMENTED** [review]
Thank you, Lesha! 💪 
You told that SA barely help us with the collaboration.
Could/should we discuss it during the standup?
In my vision, we are doing those changes with the idea in mind of enhancing the collaborative capabilities of NF.

File: .neuro/live.yml (hunk 1 above)
Comment (anayden) [reply .neuro/live.yml:31]: what about volumes? sorry, I don't undestand  the question :(

File: README.md (hunk 2 above)
Comment (anayden) [reply README.md:58]: As discussed during the standup, I'd rather see it in `neuro-extras`. This set of commands has nothing to do with neuro-flow specifically. The same sharing strategy can be applied to projects wihtout neuro-flow at all.

File: config/pipeline.json (hunk 4 above)
Comment (anayden) [reply config/pipeline.json:11]: And send one more file via base64 to reduce transparency even more?

File: README.md (hunk 3 above)
Comment (anayden) [reply README.md:60]: done

File: .neuro/live.yml (hunk 1 above)
Comment (YevheniiSemendiak) [reply .neuro/live.yml:31]: You've added `$[[ project.owner ]]` prefix into the images, but not to volumes.
I was thinking about it to keep the data and code in sync between the project runners (end users).

File: config/pipeline.json (hunk 4 above)
Comment (YevheniiSemendiak) [reply config/pipeline.json:11]: yeah, that's not a good idea..

File: .neuro/live.yml (hunk 1 above)
Comment (anayden) [reply .neuro/live.yml:31]: Oh, you're right, thanks! Pushed.

**Review by YevheniiSemendiak — APPROVED** [review]
LGTM! Thank you!

**Pull request merged**